src/pages/signup.tsx

```diff
@@ -27,13 +32,51 @@ const Wrapper = styled.section`
   }
 `;
 
+const Error = styled.p`
+  color: ${({ theme }) => theme.colors.messageText.red};
```

Comment:
Maybe rename this to something less specific. This is only one suggestion but feel free to come up with a more appropriate name.
```suggestion
  color: ${({ theme }) => theme.colors.alert.danger};
```